src/Mafia/Home.hs

```diff
@@ -62,6 +62,7 @@ installBinary package deps = do
   -- We can remove this at some point
   -- NOTE: This makes no attempt at being threadsafe
   whenM (liftIO (doesFileExist prefix)) (removeFile prefix)
+  createDirectoryIfMissing True prefix
```

Comment:
> > Unfortunately there is a slight race condition here, but we *think* it is benign.
> 
> :runs_away:

That race is technically possible in this case as well, because we used to create a file with the same name as the directory.

Very unlikely though :)